DXMainClient/DXGUI/Generic/LoadingScreen.cs

```diff
@@ -71,6 +70,18 @@ public override void Initialize()
             }
         }
 
+        protected override void GetINIAttributes(IniFile iniFile)
+        {
+            base.GetINIAttributes(iniFile);
+
+            string tmp = iniFile.GetStringValue(Name, "RandomTextures", string.Empty).Trim();
+
+            randomTextures = tmp == string.Empty ? null : tmp.Split(',');
```

Comment:
`tmp` is a bad name

GetStringValue().Split(',').Select(value=>value.Trim()).ToList();

Reply:
Check bb0e66b2d61859c39825fb6284e0638a4891b8dc.